R/robustbase-glmrob-tidiers.R

```diff
@@ -0,0 +1,114 @@
+#' @templateVar class glmrob
+#' @template title_desc_tidy_lm_wrapper
```

Comment:
Okay, I think this is a reasonable place to start, but we're discovering the "shove everything through the `lm` tidiers" approach is causing a fair amount of trouble. Let's make `tidy.glmrob()` and `augment.glmrob()` standalone functions that don't call `tidy.lm()` and `augment.lm()`.

Reply:
Sounds good!